Makefile

```diff
@@ -21,6 +21,10 @@ test-before-build:
 	! git --no-pager grep -L "^slug: " _posts
 	## Check that all slugs are unique
 	! git --no-pager grep -h "^slug: " _posts | sort | uniq -d | grep .
+	## Check for things that should probably all be on one line
+	@ ## Note: double $$ in a makefile produces a single literal $
+	! git --no-pager grep -- '^ *- \*\*[^\*]*$$'
```

Comment:
I don't think the backslash is necessary in the square brackets:

```
→ echo '*' | grep [*]; echo $?
*
0
→ echo '*' | grep [^*]; echo $?
1
```

I've verified that removing the `\` yields the same result when run against master.